packages/ses/package.json

```diff
@@ -100,6 +109,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.29.1",
+    "hermes-engine-cli": "^0.12.0",
```

Comment:
Let’s add explicit dependencies on the relevant Babel packages.

Reply:
note: added the same versions to `packages/ses` for parity

https://github.com/endojs/endo/blob/f71479f0f199e61e4d02121dbd52466a784168d8/packages/module-source/package.json#L44-L50

https://github.com/endojs/endo/blob/f71479f0f199e61e4d02121dbd52466a784168d8/packages/evasive-transform/package.json#L80-L85

as devDepencencies since they're not used in any `packages/ses/src` code
only `packages/ses/scripts/hermes-transforms.js` to bundle ses itself